avalanche/models/bic_model.py

```diff
@@ -0,0 +1,22 @@
+import torch
+
+
+class BiasLayer(torch.nn.Module):
+    """Bias layers with alpha and beta parameters"""
+
+    def __init__(self, device, clss):
```

Comment:
can you add docstrings for all the public methods? rest style, description of method and all arguments